hawkbit-rest/hawkbit-ddi-api/src/test/java/org/eclipse/hawkbit/ddi/json/model/DdiActionFeedbackTest.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.hawkbit.ddi.json.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+
+import org.assertj.core.util.Lists;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.MismatchedInputException;
+
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+
+/**
+ * Test serializability of DDI api model 'DdiActionFeedback'
+ */
+@Feature("Model Tests - Direct Device Integration API")
```

Comment:
Sorry to bother you again, but `Model Tests` are not a valid type here (see [here]( https://github.com/eclipse/hawkbit/blob/master/CONTRIBUTING.md#test-documentation)). I think `Unit Tests` fits best instead

Reply:
Sorry for the sign off. Also adjusted the naming

Reply:
No worries, thanks!